### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# ChangeLog
+
+## 0.0.1
+
+-
```
Comment (apupier) [inline CHANGELOG.md]: maybe worth mentioning the 3 extensions that are available?

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
-# vscode-datavirt-extension-pack
\ No newline at end of file
+# VS Code - DataVirt Extension Pack
```
Comment (apupier) [inline README.md]: maybe better to use the display name "Extension Pack for Data Virtualization by Red Hat"

File: CHANGELOG.md (hunk 1 above)
Comment (lhein) [reply CHANGELOG.md]: done

File: README.md (hunk 2 above)
Comment (lhein) [reply README.md]: done

**Review by apupier — APPROVED** [review]

**Pull request merged**